doc/installation_linux_manual.md

```diff
@@ -0,0 +1,143 @@
+Infrequently, apt-get mirrors or repositories may also cause timeout. For _librealsense_ users behind an enterprise firewall, \
+configuring the system-wide Ubuntu proxy generally resolves most timeout issues.
+
+**Important:** Running RealSense Depth Cameras on Linux requires patching and inserting modified kernel drivers. \
+Some OEM/Vendors choose to lock the kernel for modifications. Unlocking this capability may require modification of BIOS settings
+
+
+## Install Dependencies
+
+1. Make Ubuntu up-to-date including the latest stable kernel:
+   ```sh
+   sudo apt-get update && sudo apt-get upgrade && sudo apt-get dist-upgrade
+   ```
+2. Install the core packages required to build _librealsense_ binaries and the affected kernel modules:
+   ```sh
+   sudo apt-get install libssl-dev libusb-1.0-0-dev libudev-dev pkg-config libgtk-3-dev
+   ```
+   **Cmake Note:** certain _librealsense_ [CMAKE](https://cmake.org/download/) flags (e.g. CUDA) require version 3.8+ which is currently not made available via apt manager for Ubuntu LTS.
+3. Install build tools
+   ```sh
+   sudo apt-get install git wget cmake
+   ```
+4. Prepare Linux Backend and the Dev. Environment \
+   Unplug any connected Intel RealSense camera and run:
+   ```sh
+   sudo apt-get install libglfw3-dev libgl1-mesa-dev libglu1-mesa-dev at
+   ```
+5. Install IDE (Optional): \
+   We use [QtCreator](https://wiki.qt.io/Main) as an IDE for Linux development on Ubuntu.
+
+**Note**:
+
+* on graphic sub-system utilization: \
+*glfw3*, *mesa* and *gtk* packages are required if you plan to build the SDK's OpenGL-enabled examples. \
+The *librealsens2e* core library and a range of demos/tools are designed for headless environment deployment.
+
+* `libudev-dev` installation is optional but recommended, \
+when the `libudev-dev` is installed the SDK will use an event-driven approach for triggering USB detection and enumeration, \
+if not the SDK will use a timer polling approach which is less sensitive for device detection.
+
+
+## Install librealsense2
+
+1. Clone/Download the latest stable version of _librealsense2_ in one of the following ways:
+   * Clone the _librealsense2_ repo
+     ```sh
+     git clone https://github.com/IntelRealSense/librealsense.git
+     ```
+   * Download and unzip the latest stable _librealsense2_ version from `master` branch \
+     [IntelRealSense.zip](https://github.com/IntelRealSense/librealsense/archive/master.zip)
+
+2. Run Intel Realsense permissions script from _librealsense2_ root directory:
+   ```sh
+   ./scripts/setup_udev_rules.sh
+   ```
+   Notice: You can always remove permissions by running: `./scripts/setup_udev_rules.sh --uninstall`
+
+3. Build and apply patched kernel modules for:
+    * Ubuntu 20/22 (focal/jammy) with LTS kernel 5.13, 5.15 \
+      `./scripts/patch-realsense-ubuntu-lts-hwe.sh`
+    * Ubuntu 18/20 with LTS kernel (< 5.13) \
+     `./scripts/patch-realsense-ubuntu-lts.sh`
+
+    **Note:** What the *.sh script perform?
```

Comment:
Please remove the `*.sh script perform? `, just start with `The script above will...`

Reply:
Done